openedx/frontend-app-course-authoring
# feat: Add drag and drop support to subsections

Adds drag and drop support to reorder subsections in courses

**Screen recording**:  

https://github.com/openedx/frontend-app-course-authoring/assets/4343949/7658b705-ece0-4ecc-b847-2c45f2a2dd81


**Testing instructions**:

1. Setup instructions according to the README.
2. Checkout [this commit](https://github.com/openedx/edx-platform/commit/aaea6e5b14) on `edx-platform` (the commit was reverted due to performance issues - but the schema should remain the same even after the bugfixes)
3. `npm run test -- "CourseOutline"`
